recipe/meta.yaml

```diff
@@ -21,12 +21,12 @@ requirements:
     - {{ compiler('cxx') }}
   host:
     - netcdf-fortran
-    - libnetcdf
+    - libnetcdf >=4.7
     - hdf5
     - mpich
   run:
     - netcdf-fortran
-    - libnetcdf
+    - libnetcdf >=4.7
```

Comment:
The whole stack is not using `libnetcdf >=4.7` yet. That is fine here but we should probably start thinking about a migration.